### ISSUE TITLE
Directory jumping priority

### ISSUE THREAD

**Issue by atwok** [marlonrichert/zsh-autocomplete#71]
### Is your feature request related to a problem? Please describe.

Hi.  I'm trying out autocomplete.   As part of that I installed zoxide and copied your cfg to try it out.  One issued I've run into is I type "emacs xyz<hit_the_tab_key>" and I'm expecting a file name expansion (because there are "xyz_*" files in the current dir) but what I get instead is "emacs /a/bunch/of/dirs/xyz/whatever/" because there is a zoxide match available.

### Describe the solution you'd like

I think if there is something 'local' that matches the expansion then it should win...?  I'm not super tied to this because I'm new.  Just throwing out ideas.  :)

### Describe alternatives you've considered

Initially I was going to suggest a switch that preferred non-zoxide/fasd/etc expansion over the other choices (whatever they are called) because some people might want one over the other, but maybe choosing the "local" match works.  Otherwise its <tab> for my preferred match as configured and <tab+some_other_key> if you want to choose from the other category on purpose.

### Additional context

if this is already controllable or selectable, I apologize for opening this issue.

**Comment by marlonrichert** [comment]
@atwok Can you please check out the `dev` branch and try out the completion behavior there? I’ve made some changes.

**Comment by marlonrichert** [comment]
@atwok Should be fine on `master` now. Reopen if not.

**Comment by atwok** [comment]
It works better now, thanks!

Just one related suggestion.  In my case, like yourself, I have 'cd' aliased to zoxide.  I was wondering if you can detect when the user is using one of these directory jumping tools and only offer them directory completions?  Or even if a user is just using regular 'cd'?  It does make sense when using other tools you want want a directory completion to get to a file.

**Comment by marlonrichert** [comment]
`zsh-autocomplete` offers frecent dirs from `zoxide` _anywhere_ that completion could offer a directory. Just try, for example, `cp`, `mv` or `rm`.

Or am I misunderstanding what you want? Could you offer some examples?

**Comment by atwok** [comment]
My ask was a bit wordy.

I was wondering if there's a way to detect the user is trying to change directory and ONLY offer frecent dirs or local dirs?

It might not be possible or worth doing.  Just thinking out load.

**Comment by marlonrichert** [comment]
You mean, instead of offering local dirs? Why would you want that? Then it would nearly impossible to navigate into local dirs. 🙂

Again, if you could give a concrete example, that would be really helpful.

**Comment by atwok** [comment]
I meant to say "ONLY offer frecent dirs or local dirs?"

So right now
> cd 
frecent_dir(s)
local_dir1/  local_dir1/
local_file.1  local_file.2

Is there some way to hide the local files when you know the user is changing directories?

Again, it's not a huge deal.  Sorry to be confusing.  :/

**Comment by marlonrichert** [comment]
Oh, that’s actually a problem with `zoxide`. You’ll see that even without `zsh-autocomplete`, when you use `zoxide` to change dirs, its completion will offer files. Please file a bug at https://github.com/ajeetdsouza/zoxide/issues
